### DISCUSSION TITLE
every run create new builder folder with huge amount of cache

### DISCUSSION THREAD

**Discussion by kamilgajekdxc** [docker/setup-buildx-action#308] [Q&A]
Hello
I noticed that when using action a new instance is generated each time (buildx_buildkit_builder folder)
The next task docker/build-push-action@v5 builds a docker image in this location and downloads a lot of other files needed when building docker.

The problem is that the next run builds a new folder and downloads the same data again.
Is it possible to:
All data from a specific folder (entire folder) was deleted after the image was finished building? Currently, post job deletes the builder, but I have the impression that it only deletes a certain part of the data?
Force the builder from the previous run to be used and maybe the downloaded files will be used instead of downloading new ones.

**Comment by kamilgajekdxc** [comment]
Other task clean all information about docker process and clean remove nothing, but also not log any error.

**Comment by crazy-max** [comment]
> Currently, post job deletes the builder, but I have the impression that it only deletes a certain part of the data?

Post step of setup-buildx-action removes the builder and buildkit state as well. If this is not the case please open an issue with repro.
